src/core/control/jobs/RenderJob.h

```diff
@@ -11,6 +11,7 @@
 
 #pragma once
 
+#include <cairo/cairo.h>  // for cairo_surface_t
```

Comment:
Is it not possible to forward declare this?

Reply:
Except cairo_surface_t is typedef'd in cairo.h. I don't know how to forward declare a typedef.